src/extension.ts

```diff
@@ -202,7 +202,7 @@ export class Extension {
       this._diagnostics,
       this._treeItemObserver,
     ];
-    await this._rebuildModel(false);
```

Comment:
Why did this change?

Reply:
This function gets called when the extension gets activated. For the initial activation we want to show the error messages, what this boolean is about: `showWarnings: true`.